src/mpyl/cli/build.py

```diff
@@ -154,46 +157,40 @@ def status(obj: CliContext):
 
 def __print_status(obj: CliContext):
     run_properties = RunProperties.from_configuration(obj.run_properties, obj.config)
-    ci_branch = run_properties.versioning.branch
-
     branch = obj.repo.get_branch
     main_branch = obj.repo.main_branch
     tag = obj.repo.get_tag if not branch else None
+    console = obj.console
 
     if not tag:
-        if ci_branch and not obj.repo.get_branch:
-            obj.console.print("Current branch is detached.")
+        if run_properties.versioning.branch and not obj.repo.get_branch:
+            console.print("Current branch is detached.")
         else:
-            obj.console.log(
+            console.log(
                 Markdown(
                     f"Branch not specified at `build.versioning.branch` in _{DEFAULT_RUN_PROPERTIES_FILE_NAME}_, "
                     f"falling back to git: _{obj.repo.get_branch}_"
                 )
             )
 
         if branch == main_branch:
-            obj.console.log(f"On main branch ({branch}), cannot determine build status")
+            console.log(f"On main branch ({branch}), cannot determine build status")
             return
 
     version = run_properties.versioning
     revision = version.revision or obj.repo.get_sha
     base_revision = obj.repo.base_revision
-    obj.console.print(
+    base_revision_specification = (
+        f"at `{base_revision}`"
+        if base_revision
+        else f"not present. Earliest revision: `{obj.repo.root_commit_hex}` (grafted)."
+    )
```

Comment:
Does this check make sense for tag builds?

Reply:
Good point. Will try to make the distinction more explicit.